app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -37,11 +37,6 @@ public void attach() {
     }
 
     @Override
-    public void detach() {
```

Comment:
@iamareebjamal Well, let it be attached for now. Once we add **LeakCanary** we can raise it up as a separate issue.